masterlist.yaml

```diff
@@ -185,6 +185,7 @@ globals:
  
 
 plugins:
+### Official Update & DLC plugins
   - name: 'Update.esm'
```

Comment:
Same separators were in F04 master-list. Thought it would useful for when groups are added.